package.json

```diff
@@ -62,14 +62,18 @@
     "glob": "^7.1.2",
     "husky": "^0.14.3",
     "inquirer": "^4.0.1",
-    "jest": "^21.2.0",
-    "jest-cli": "^21.2.1",
+    "jest": "^22.0.4",
```

Comment:
I added these dependencies to make sure the right versions were hoisted to the top level.. It seems CRA has some old version of something